packages/internal/src/build/babel/web.ts

```diff
@@ -42,6 +48,8 @@ export const getWebSideBabelPlugins = (
       require('../babelPlugins/babel-plugin-redwood-src-alias').default,
       {
         srcAbsPath: rwjsPaths.web.src,
+        // adds the paths from [ts|js]config.json to the module resolver
+        ...getPathsFromConfig(config.web),
```

Comment:
I don't think this will do anything; `babel-plugin-redwood-src-alias` is an in-house Babel plugin that re-writes `./src`. It only takes one option, `srcAbsPath`:

https://github.com/redwoodjs/redwood/blob/88f68ef867db29327c0848f9bed5e89435775683/packages/internal/src/build/babelPlugins/babel-plugin-redwood-src-alias.ts#L7-L10 

```suggestion
```